### DISCUSSION TITLE
How do i view my previous accepted solutions

### DISCUSSION THREAD

**Discussion by sumukhvarma21** [codewars/codewars.com#3227] [Q&A] [answered]
I revisited a prolem i solved a couple months ago and want to see the solution i submitted at that point in time 
How to do this?

**Comment by esh3081** [comment, marked as answer]
![image](https://github.com/codewars/codewars.com/assets/170317252/fd1506da-c9a2-4095-8b67-37835a2ebbdf)

I guess, you can go to the kata => Click on Solutions tab => then pick a Filter "Mine"

**Reply by sumukhvarma21** [reply]
Thank you
